cluster/Vagrantfile

```diff
@@ -30,7 +30,7 @@ node_os = ENV['CONTIV_NODE_OS'] || CENTOS
 k8s_ver = ENV['CONTIV_K8S_VERSION'] || DEFAULT_K8S_VERSION
 orc_path = case k8s_ver
            when /^v1\.[45]\./ then 'k8s1.4/'
-           when /^v1\.6\./    then 'k8s1.6/'
+           when /^v1\.[67]\./ then 'k8s1.6/'
```

Comment:
Is the behavior / working of k8s 1.6 and 1.7 similar ? 
We are outputting to k8s.16 even if k8s version is 1.7.